content/docs/user-guide/basic-concepts/workspace.md

```diff
@@ -6,3 +6,40 @@ tooltip: >-
   models, etc. Typically, it's also a Git repository. It will contain your DVC
   project.
 ---
+
+<!-- keywords: data science project architecture, machine learning project architecture, machine learning workflow, data science workflow, machine learning file system, data science file system, data science project structure, machine learning project structure, notebook version control -->
+
+# Workspace
+
+A data science project consists of data obtained from many different sources.
+This data may be split into multiple files or directories or (as the project
+structure needs) have different versions for different requirements. e.g. A
+smaller / simplified version might be required in prototyping for faster
+feedback and shorter training times. A single workspace to manage all artifacts
+of a project is desirable, although versioning needs and managing dependencies
+make it increasingly complex.
+
+DVC allows a single directory to contain all your project artifacts. In the
```

Comment:
What's the main point of this part of the story/ motivation? I'm not sure what we want to say here. I'm starting to think "workspace" is kind of a plain concept which may not need much introduction at all.

I know in part you're trying to fit a bunch of SEO words but you were right that those are secondary. Let's get the story right first I think, and then sprinkle special terms if possible.

Reply:
I think that mentioning directory structures could definitely make sense. Just not sure whether that comes before or after defining what the workspace is. Probably after in this case?

Reply:
Directory structure of the workspace or `.dvc/` I think a separate _DVC project_ (or repository) BC document is more suitable for this.

Reply:
No, I mean "data may be split into multiple files or directories" etc. — how to structure files in the workspace dir.

Reply:
p.s. there's an entire guide for the [spec of .dvc/](https://dvc.org/doc/user-guide/project-structure/internal-files) already, no need to add that to the project concept page 🙂

Reply:
I can tell these in a "there are such and such options to organize your data and models" way I think. It looks too context/project dependent to have a clear recommendation. Bottom line: User can organize their data in any way they like 😄 

Is there a best practice document I can take a look at for this? I can probably cook some set of examples, is this what you would like?

Reply:
> too context/project dependent to have a clear recommendation... organize their data in any way they like

OK. So why did you include that info in the first place? "data may be split into multiple files or directories..."

> best practice document I can take a look at for this? I can probably cook some set of examples

Let's please focus on this issue (write the workspace concept page) for now. There's a separate epic issue for Best Practices somewhere but again, let's not go there now 🙂

Reply:
A data science project can have different kinds of data, from different sources, in different directories distributed in different machines. Workspace is where we collect them together. I mean this. 

Could you please check the changes in 0b099f6. Thank you 😄

Reply:
OK. I see you rewrote the 2nd paragraph. This comment refers to the first paragraph, the motivation/intro.

> A data science project can have different kinds of data, from different sources, in different directories distributed in different machines.

OK that's clear and concise. Why not use something direct and short like that for the motivation part? May not even need a whole paragraph 🙂

As it is now you need to read a whole paragraph and then another opening sentence (in the next p) to get to the "what is workspace" concept definition.

Again, just trying to filter out information that may be distracting/confusing/repetitive, even all the details everything you're writing is important in other contexts. If editing current text is becoming difficult I recommend you put this on a side and start over, then compare both versions if needed. Thanks